Constants.h

```diff
@@ -39,6 +39,7 @@ namespace PlayerConst
 	constexpr float SPEED = 100;
 	constexpr int BOMB_COUNT = 3;
 	constexpr int BOMB_LENGTH = 3;
+	constexpr int LIFES = 3;
```

Comment:
Thoughts  on naming it `LIFES_COUNT`?
And also, we could rename `BOX_COUNT` to `BOXES_COUNT` and `BOMB_COUNT` to `BOMBs_COUNT`

Reply:
done